recipes/bob.learn.boosting/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "2.0.7" %}
+
+package:
+  name: bob.learn.boosting
+  version: {{ version }}
+
+source:
+  fn: bob.learn.boosting-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.learn.boosting/bob.learn.boosting-{{ version }}.zip
+  md5: d02c5be0c59f55587409a558a98bf04e
+
+build:
+  entry_points:
+    - boosting_example.py = bob.learn.boosting.examples.mnist:main
+  number: 0
+  skip: true  # [not linux]
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - scipy
+    - bob.extension
+    - bob.blitz
+    - bob.core
+    - bob.io.base
+    - cmake
+    - pkg-config
+    - gcc     # [linux]
```

Comment:
So, there is a compiler on Linux so if you don't need this don't add it. Our plan is to make this unnecessary in the future. Even for C++11 support (possibly C++14 too). Though I thought we had a discussion with one of bob's other developers and determined we might need this. I don't remember which one it was one though. Maybe bob.math or something like that.